### ISSUE TITLE
Missing Metabase features

### ISSUE THREAD

**Issue by slvrtrn** [ClickHouse/metabase-clickhouse-driver#183]
During the last year of releases, we were mainly focused on addressing the breaking changes affecting the driver and fixing bugs.

However, we were not as focused on implementing the newly introduced Metabase _driver features_ until now. I'd like to address this, starting from #179, which is included in the latest 1.2.0 release that is intended to be used with Metabase 0.47.x.

If you have a particular feature that you are missing in the driver, please don't hesitate to react and leave a comment here, so this issue can be used as a source for ideas on what should be implemented next.

**Comment by brendavarguez** [comment]
Hello @slvrtrn. Something I recently discovered is that for date columns of specific type `Nullable(Date)` I can easily perform aggregations (count, sum, avg, ...) by date truncated by day, week, month, etc. However, for date columns of specific type `Nullable(DateTime('America/Mexico_City'))` I'm not having the same options. When building queries from the UI, I can only make aggregations by date truncated by day (seems like it's the default).

My company is migrating from Postgres --> ClickHouse and having the timezone + making aggregations it's important to us since many reports are build using Metabase.

Not sure it it's related to #182

**Comment by slvrtrn** [comment]
@brendavarguez, this should be fixed as of [1.2.0](https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.2.0) with Metabase 0.47. It was related to #178.

**Comment by brendavarguez** [comment]
@slvrtrn [latest](https://github.com/metabase/metabase/releases) seems to be 0.46.6.4

**Comment by slvrtrn** [comment]
@brendavarguez, yes, 1.2.0 is for the [upcoming 0.47],(https://github.com/metabase/metabase/milestone/172) which is almost there.

**Comment by alekseyp** [comment]
We've tried to switch from PostgreSQL to ClickHouse and found couple issues so far:

- nested-field-columnsa: we use a lot of jsonb which was converted to string in ClickHouse, but so far we haven't figured out how can we use our json data the same way it's working between Metabase and pgsql
- convert-timezone: this one is still WIP, but as far as I know we store everything in UTC and some reports are in diff timezone, those work with pgsql, but I see reports that we have hard time make them work same way with ClickHouse.
